### DISCUSSION TITLE
Getting started with the API

### DISCUSSION THREAD

**Discussion by jmeisenthal** [bpepple/metron#93] [Q&A] [answered]
Hi @bpepple 
Thanks for creating Metron! Could you point me to any docs you have for getting started with using the Metron web API?

Thanks!

**Comment by bpepple** [comment, marked as answer]
You can find the docs for the API here: https://metron.cloud/docs/

There's also a python library to use it: https://github.com/bpepple/mokkari

If you have any questions, don't hesitate to ask.
